doc/api/next_api_changes/removals/27095-AL.rst

```diff
@@ -0,0 +1,5 @@
+Inconsistent *nth_coord* and *loc* passed to ``_FixedAxisArtistHelperBase``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Passing an inconsistent *nth_coord* and *loc* (e.g., requesting a "top y axis"
+or a "left x axis") to ``_FixedAxisArtistHelperBase`` or its subclasses is now
+an error.
```

Comment:
Given that _nth_coord_ is now ignored, is it true that this is an error?

Reply:
fair enough, reworded.